src/pages/MainPage/RouteData.js

```diff
@@ -20,15 +23,15 @@ function NoMatch() {
 const RouteData = () =>{
     return(
         <Routes>  
-           {/* <Route exact path='/user' element={<PrivateRoute/>}>                
-              <Route exact path="/user/liked-videos"  element={<LikedVideosPage/>}/> 
-              <Route exact path='/user/playlists' element={<PlayList/>}/>
-              <Route exact path='/user/watchlater' element={<WatchLater/>}/>
-              <Route exact path='/user/history' element={<History/>}/>
+           <Route exact path='/user' element={<PrivateRoute/>}>                
+              {/* <Route exact path="/user/profile/:id"  element={<UserProfile/>}/>  */}
```

Comment:
here you are doing code splitting on route base level. but you haven't wrapped component under suspense
eg:

`**<suspense fallback={//fallback component goes here}>//component goes here</suspense>**`

Reply:
will follow the above mentioned structure 
![image](https://user-images.githubusercontent.com/76782963/170124585-1dd42417-1e1a-4d53-a3ec-c5165096c1c0.png)